### PR TITLE
SDA-3966 Support Paste as plain text

### PR DESCRIPTION
**Pull request by sbenmoussati**
## Description
Support `Paste as plain text`

### CODE REVIEW

File: src/app/context-menu-builder.ts [hunk 1]
```diff
@@ -433,6 +435,21 @@ export class ContextMenuBuilder {
     return menu;
   }
 
+  /**
+   * Adds the Paste menu item.
+   */
+  public addPasteAndMatchStyle(menu, menuInfo) {
```
Comment (NguyenTranHoangSym) [inline src/app/context-menu-builder.ts:441]: should this be renamed into addPasteAsPlainText? since the meaning of whole thing is for doing this

Comment (sbenmoussati) [reply src/app/context-menu-builder.ts:441]: depends if we're considering chromium definition or functional def.

File: src/app/context-menu-builder.ts [hunk 2]
```diff
@@ -433,6 +435,21 @@ export class ContextMenuBuilder {
...
+  /**
+   * Adds the Paste menu item.
+   */
+  public addPasteAsPlainText(menu, menuInfo) {
+    menu.append(
+      new MenuItem({
+        label: this.stringTable.pasteAsPlainText(),
+        accelerator: 'CommandOrControl+Shift+V',
+        enabled: menuInfo.editFlags.canPaste,
+        role: 'pasteAndMatchStyle',
```
Comment (NguyenTranHoangSym) [inline src/app/context-menu-builder.ts:447]: is this suppose to be pasteAsPlainText?

Comment (sbenmoussati) [reply src/app/context-menu-builder.ts:447]: yes, which correspond to role "pasteAndMatchStyle" on chromium

**Review by NguyenTranHoangSym — APPROVED** [review]

**Pull request merged**